sysutils/git-backup/src/opnsense/mvc/app/library/OPNsense/Backup/Git.php

```diff
@@ -117,7 +117,7 @@ public function setConfiguration($conf)
     }
 
     /**
-     * Backup is responsible for initialising the local repo and pusing it to upstream.
+     * Backup is responsible for initialising the local repo and pushing it to upstream.
```

Comment:
this shouldn't be in this PR, but it's a nice change to have indeed :)

Reply:
please see #4345